4ChanWebScraper.py

```diff
@@ -31,16 +31,24 @@ def createFolderName(webpageTitle):
     outputString = splitString[0].strip().upper() + ' - ' + normalize_whitespace(splitString[1]).lower().title()
     return outputString
 
+def validateUrl(url):
+    pattern = r'^(https?://)?(www\.)?4chan'
```

Comment:
actually it won't fail on full URLs, but there is a problem with it.  as it is this would accept something like, "4chanaisreallycool.org" as a domain because we're not searching for the domain separating period.  i'll fix that.